### DISCUSSION TITLE
Disable rendering of URLs as links

### DISCUSSION THREAD

**Discussion by florianjung79** [sibiraj-s/ngx-editor#367] [Q&A] [answered]
Is it possible to disable rendering URLs as links?

Given the following use case:

I have an editor instance configured to exclude linking and unlinking of plain text (thus the toolbar icons are not shown). To prevent having HTML links being pasted into the editor I added a plugin that sanitized every content pasted. This works quite well as I see the plugin invocation and the sanitized content that is returned from the plugin. As soon as the content (which contains text that matches URL patterns) is processed by the editor itself, the (once plain) URL text content is transformed back into a link.

Is there any chance to disable the editor's behaviour as it re-introduces interpreted content that was once sanitized? Any help is appreciated.

**Comment by sibiraj-s** [comment, marked as answer]
You can now disable linking on paste by

```ts
new Editor({
  features: {
    linkOnPaste: false
  }
})
```

Added in v11.1.0

[a comment by an automated account is not shown]
